tests/unit/models/test_base_model.py

```diff
@@ -597,3 +604,415 @@ def test_to_xrpl_signer(self):
             ],
         }
         self.assertEqual(tx.to_xrpl(), expected)
+
+    def test_to_xrpl_amm_create(self):
```

Comment:
I don't actually think it's useful to have a test for each one of these - just make sure there's a test or two that cover the new types.

Reply:
(especially if there are integration tests)

Reply:
You'll need to keep at least one to test the `AuthAccounts` fix though - otherwise there won't be any tests checking that.